eng/validate-code-formatting.ps1

```diff
@@ -0,0 +1,23 @@
+param (
+    [string]$rootDirectory,
+    [string[]]$includeDirectories
+)
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+try {
+  . (Join-Path $PSScriptRoot "build-utils.ps1")
+  Push-Location $RepoRoot
+
+  $dotnet = Ensure-DotnetSdk
+  Exec-Console $dotnet "tool run dotnet-format -v detailed whitespace $rootDirectory --folder --include-generated --include $includeDirectories --verify-no-changes"
+
+  exit 0
```

Comment:
Why did you use `exit` here vs. `ExitWithCode`?

Reply:
Was following the generate-compiler.code.ps1 patterns https://github.com/dotnet/roslyn/pull/71666/files#diff-64ff4bf5168c08ddeb40d75a8889a9ad53f4f2bf3ff1b320a320cdf3567227c0R139